config/conf.d/global.conf

```diff
@@ -0,0 +1,9 @@
+global: {
+    # Path to logback logging configuration file; for more
+    # info, see http://logback.qos.ch/manual/configuration.html
+    logging-config: /etc/puppetserver/logback.xml
+}
+
+product: {
+    name: puppetserver
+}
```

Comment:
Just FYI this directory is configurable in `lein-ezbake` now, I just haven't changed it in this PR yet. So are you thinking `ezbake/config` would be good?